synapse/cortex.py

```diff
@@ -916,6 +916,15 @@ async def getPermDefs(self):
         '''
         return await self.cell.getPermDefs()
 
+    async def getAxonUpload(self):
+        self.user.confirm(('axon', 'push'))
```

Comment:
the existing perm for this is`axon.upload`, not sure if this was intended to be different.